vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/retrievable/attributes/time/TimestampAttribute.kt

```diff
@@ -0,0 +1,16 @@
+package org.vitrivr.engine.core.model.retrievable.attributes.time
+
+import org.vitrivr.engine.core.model.retrievable.attributes.RetrievableAttribute
+import java.util.concurrent.TimeUnit
+
+/**
+ * A [RetrievableAttribute] encoding a timestamp.
+ *
+ * @author Ralph Gasser
+ * @version 1.0.0
+ */
+data class TimestampAttribute(val timestampNs: Long) : RetrievableAttribute, Comparable<TimestampAttribute> {
```

Comment:
If the other one is called `TimerangeAttribute`, I'd argue this one should be called `TimePointAttribute`

Reply:
Done!